insitu/models.py

```diff
@@ -103,7 +161,7 @@ def __str__(self):
         return self.name
 
 
-class ProductRequirement(models.Model):
+class ProductRequirement(SoftDeleteModelMixin):
```

Comment:
Change the naming - mix-in means extra behaviour on something existent, so a class can never inherit from mix-in. Maybe SoftDeleteModel?